resolver/resolver.cc

```diff
@@ -2542,12 +2542,13 @@ class ResolveSignaturesWalk {
         auto methodInfo = method.data(ctx);
 
         // Is this a signature for a method defined with argument forwarding syntax?
-        if (methodInfo->arguments().size() == 3) {
+        if (methodInfo->arguments().size() >= 3) {
             // To match, the definition must have been desugared in exaclty 3 parameters named
             // `<fwd-args>`, `<fwd-kwargs>` and `<fwd-block>`
```

Comment:
Since the condition has changed, we should update this comment, something like "To match, the last three parameters of the definition must be named ..."